plugins/module_utils/foreman_helper.py

```diff
@@ -132,6 +132,39 @@ def _patch_location_api(self):
         _location_update_params_location = next(x for x in _location_update['params'] if x['name'] == 'location')
         _location_update_params_location['params'].append(_location_organizations_parameter)
 
+    def _patch_sync_plan_api(self):
+        """This is a workaround for the broken sync_plan apidoc in katello.
+            see https://projects.theforeman.org/issues/27532
+        """
+
+        _organization_parameter = {
+            u'validations': [],
+            u'name': u'organization_id',
+            u'show': True,
+            u'description': u'\n<p>Filter sync plans by organization name or label</p>\n',
+            u'required': False,
+            u'allow_nil': False,
+            u'allow_blank': False,
+            u'full_name': u'organization_id',
+            u'expected_type': u'numeric',
+            u'metadata': None,
+            u'validator': u'Must be a number.',
+        }
+
+        try:
+            _sync_plan_methods = self.foremanapi.apidoc['docs']['resources']['sync_plans']['methods']
+
+            _sync_plan_add_products = next(x for x in _sync_plan_methods if x['name'] == 'add_products')
+            if next((x for x in _sync_plan_add_products['params'] if x['name'] == 'organization_id'), None) is None:
+                _sync_plan_add_products['params'].append(_organization_parameter)
+
+            _sync_plan_remove_products = next(x for x in _sync_plan_methods if x['name'] == 'remove_products')
+            if next((x for x in _sync_plan_remove_products['params'] if x['name'] == 'organization_id'), None) is None:
+                _sync_plan_remove_products['params'].append(_organization_parameter)
+        except Exception:
```

Comment:
in #392 I patched that under KatelloApypieModule, as sync plans are a Katello thing.